Zyantific.Zydis/Generated/EnumISAExt.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
```

Comment:
These imports are not needed (same for all generated enum files, so maybe we should remove them from the generator @athre0z)